calcom/cal.com
# Feat: v2 app store category/categories

## What does this PR do?

Implement the v2 UI for a category or all categories.
Plus some changes to app store page that had design updates/definitions.

Fixes #3778 depends #3911 as the branch was based on it.

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Go to categories page at `/apps/categories` or a single one at `/apps/categories/video` and see the new App Cards.

> **Note that the cards don't have the "Install" button on them, as the implementation prioritized showing the page quickly rather than converting these pages from static to SSR to be able to get existing `credentials` for each app for the logged in user in order to efficiently show the installation status and button.**
